api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -580,6 +580,18 @@ private boolean userHasEverLoggedIn(
       // See RW-1488.
       throw new BadRequestException("Changing email is not currently supported");
     }
+    updateInstitutionalAffiliations(updatedProfile, user);
+
+    // This does not update the name in Google.
```

Comment:
Can you comment on the semantics/lifecycle keeping these in sync, and why we don't need to here?

Reply:
I know nothing about this.  I just moved it.

Reply:
We need to be careful moving the audit stuff like `fireUpdateAction()`. This looks right though. Can you please verify we still get the events written to the audit stream?

Reply:
What I moved was the `updateInstitutionalAffiliations()` block into its own method.  Everything else remained in the same location.